### ISSUE TITLE
gh issue template

### ISSUE THREAD

**Issue by e-marshall** [dcherian/tidy-xarray#7]
Create an issue template to solicit examples from community

**Comment by scottyhq** [comment]
Agreed @e-marshall this would be fun! perhaps this repository should live under the xarray-contrib org or pangeo-data? 

Here is another recent one I came across helping a colleague. https://gist.github.com/scottyhq/cb2c0f412b67103457529b76a76de289

I think it illustrates a common case of netCDFs with minimal metadata, so you have to add that context. It could be worth highlighting what I think is a helpful pattern when working with a new dataset (which I've tried to capture below thinking these points could be template headings):

1. know your use case: are you interested in time series? spatial analysis? both?  in this case time-series at specific points, but the data files are partitioned into 1x1 degree zones with 1 file per year over several decades.
2. inspect a single file. is it tidy? in this case we have non-dimensional coordinates that need to be promoted to dimensional-coordinates for selection and alignment/concatenating in time. The coordinate values (stats) are described in an external PDF
    - there is critical information encoded in the file name rather than in attributes of the files, so that needs extracting
    - also be sure to inspect the 'encoding' for on-disk chunks, since our workflow requires open_mfdataset we are using dask and therefore should make sure our workflow uses some multiple of the on-disk chunks (https://docs.dask.org/en/latest/array-best-practices.html)
3.  transfer what we've done to a single file to the 'preprocess' function of open_mfdataset so that we are able to construct a nice datacube for analysis (and benefit from expressive selection and grouping and other features of the Xarray API)
